CONTRIBUTORS

```diff
@@ -121,6 +123,7 @@ Jacob Davis <jacobdavis11@gmail.com>
 Jacob Li Peng Cheng <lipech@hotmail.com>
 Jakub Osika <osika.jakub@gmail.com>
 James James John <jamesjay4199@gmail.com>
+James Xu <jamesxu@umich.edu>
```

Comment:
Are we expecting a similar change in the AUTHOR file as well?

Reply:
Nope, this is correct as written. Thanks for checking!